src/sth_configuration.js

```diff
@@ -0,0 +1,38 @@
+/* globals module, process */
+
+(function() {
+  "use strict";
+
+  var ENV = process.env;
```

Comment:
Absolutely right, `process.env` is a Javascript object including a property for each environment variable passed when the Node.js application is executed. The key of each one of these properties are the environment variable names and their values are the values of these environment variables.
For example: VAR1=VALUE1 VAR2=VALUE2 node app.js
In app.js there exists a process.env object of the following shape:
```json
{
  "VAR1": 'VALUE1",
  "VAR1": "VALUE2"
}
```